### ISSUE TITLE
Java project does not run anymore when using redhat.java

### ISSUE THREAD

**Issue by lmcbout** [eclipse-theia/theia#6791]
### Description
Cannot run Theia using redhat.java. Replace the @theia/java and @theia/java-debug in package.json with the java version below. Used to work up to  mid last week. When starting a workspace for a JAVA project compiled with/without maven We have now the following message:
"The Language Support for Java server crashed 5 times in the last 3 minutes. The server will not be restarted."

### Reproduction Steps
Create a HelloWorld.java 
Open the HelloWorld.java file, you should have the Run | Debug over the main class. Does not show anymore
See my-app.tar.gz as a maven project attached

**OS and Theia version:**
Theia:latest
Ubuntu 16.04
redhat.java-0.54.2.vsix
vscjava.vscode-java-debug-0.23.0.vsix
vscjava.vscode-java-dependency-0.6.0.vsix
vscjava.vscode-java-test-0.22.0.vsix
vscjava.vscode-maven-0.20.1.vsix

**Diagnostics:** 

https://github.com/eclipse-theia/theia/files/3996059/my-app.tar.gz

**Comment by vince-fugnitto** [comment]
I believe when you added the command line argument to start it still did not work, correct?

```
--vscode-api-version=1.40.0
```

**Comment by lmcbout** [comment]
You are correct @vince-fugnitto . Even with the option on the command line when starting : yarn start ...

**Comment by akosyakov** [comment]
Probably depends on https://github.com/eclipse-theia/theia/issues/6804?
